.editorconfig

```diff
@@ -14,3 +14,9 @@ tab_width = 4
 [{*.json, *.yaml, *.yml, *.md}]
 indent_style = space
 indent_size = 2
+
+[tests/fixtures/files/interactive.json]
```

Comment:
Proc je tu tohle?

Reply:
Protože IDE mi neustále přepisoval soubor a padaly testy. Máš jiný nápad na opravu?

Reply:
Jak ti to prepisovalo testy? A jak padaly?

Reply:
![obrazek](https://user-images.githubusercontent.com/6066243/148379969-ab59bcbe-1ec8-44df-9ef4-5e0805129028.png)

[output.zip](https://github.com/contributte/neonizer/files/7821551/output.zip)